cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -994,3 +1045,64 @@ def _revert():
 
     view = navigate_to(ansible_service_funcscope, "Details")
     assert view.provisioning.credentials.get_text_of("Cloud") == provider_credentials.name
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1505929])
+def test_service_ansible_service_name(request, appliance, dialog_with_catalog_item):
+    """
+    After creating the service using ansible playbook type add a new text
+    field to service dialog named "service_name" and then use that service
+    to order the service which will have a different name than the service
+    catalog item.
+
+    Bugzilla:
+        1505929
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/2h
+        tags: ansible_embed
+    """
+    # Extracting Ansible Catalog Item and Element name from fixture.
+    ansible_cat_item, ele_name, ansible_catalog = dialog_with_catalog_item
+
+    # Navigate to Service Catalog order page.
+    service_catalogs = ServiceCatalogs(
+        appliance, ansible_catalog, ansible_cat_item.name)
+    view = navigate_to(service_catalogs, 'Order')
+
+    # Fill the different than the dialog name and order service.
+    service_name = "different_service_name"
+    view.fields(ele_name).fill(service_name)
+    time.sleep(5)
+    view.submit_button.click()
+
+    # Service name is updated so ansible_service_request will not work.
+    request_descr = (
+        f"Provisioning Service [{ansible_cat_item.name}] from [{ansible_cat_item.name}]"
+    )
+    service_request = appliance.collections.requests.instantiate(description=request_descr)
+    service_request.wait_for_request()
+
+    # Tear down for service_request and service with different name.
+    @request.addfinalizer
+    def _revert():
+        if service_request.exists:
+            service_request.wait_for_request()
+            service_id = appliance.rest_api.collections.service_requests.get(
+                description=request_descr
+            )
+            appliance.rest_api.collections.service_requests.action.delete(
+                id=service_id.id
+            )
+        if service.exists:
+            service.delete()
+
+    # Go to Ordered service page and assert new name of service.
+    service = MyService(appliance, service_name)
+    view = navigate_to(service, "Details")
```

Comment:
If my intention is only to assert service with name provided in dialog is created or not then I would just check `exists`